examples/CMakeLists.txt

```diff
@@ -50,7 +50,7 @@ if(GINKGO_HAVE_PAPI_SDE)
 endif()
 
 if(GINKGO_BUILD_MPI)
-    list(APPEND EXAMPLES_LIST distributed-solver)
+    list(APPEND EXAMPLES_LIST distributed-solver distributed-preconditioned-solver)
```

Comment:
I would be fine with merging the two examples. I don't see much value in having them separated.

Reply:
I have merged them.